hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalWanPublisherStatsImpl.java

```diff
@@ -38,6 +38,7 @@
 
     private volatile boolean connected;
     private volatile boolean paused;
+    private volatile boolean stopped;
```

Comment:
Shouldn't we track the status here with a `volatile WanPublisherState` instead? State stopped intrinsically means replication is paused as well, but this class allows to set `paused=false` while `stopped=true`.

Reply:
Unfortunately, the stats are public API so we might add a `getState` method but we need to keep the `isPaused` method.

Reply:
I meant only the fields, not changing the API. My only point here is to make sure that `paused==true` if `stopped==true`. So changing isPaused() to something like this:
```
public boolean isPaused() {
  return replicationState.isReplicateEnqueuedEvents();
}
```